client/src/containers/AboutUsContainer/index.js

```diff
@@ -1,12 +1,79 @@
 import React, { Component } from 'react';
+import cat1 from '../../assets/cat-1.jpeg';
+import cat2 from '../../assets/cat-2.jpeg';
+import cat3 from '../../assets/cat-3.jpeg';
+
+const styles = {
+  AboutUsContainer: {
+    border: '10px solid blue'
+  },
+  header: {
+    paddingLeft: '3rem',
+    paddingTop: '3rem',
+    marginTop: '5rem',
+    fontFamily: 'Roboto',
+    fontSize: '3rem',
+    fontWeight: 'bold',
+  },
+  headerSubText: {
+    fontSize: '1.7rem',
+    fontStyle: 'italic',
+    paddingLeft: '3rem',
+  },
+  introText: {
+    fontSize: '1.2rem',
+    paddingLeft: '3rem',
+    paddingBottom: '2rem',
+    fontFamily: 'Merriweather'
+  },
+  h2: {
+    fontFamily: 'Roboto',
+    lineHeight: '3rem'
+  },
+  aboutBody: {
+    paddingLeft: '3rem',
+    paddingRight: '3rem',
+    fontFamily: 'Merriweather',
+    lineHeight: '1.6rem'
+  },
+  imageBar: {
+    display: 'flex',
+    justifyContent: 'center',
+    border: '10px solid red',
+    flexWrap: 'wrap'
+  }
+}
 
 class AboutUs extends Component {
-  
+
   render() {
     return (
-      <div>
-      <h1>About Us</h1>
-      <p></p>
+      <div style={styles.AboutUsContainer}>
+        <div style={styles.header}>
+          <h1>About Us</h1>
+        </div>
+        <div style={styles.headerSubText}>
+          <p>Giving you better odds through data science, machine learning, and years of football fandom.</p>
+        </div>
+        <div style={styles.headerSubText}>
+          <p>----------</p>
+        </div>
+        <div style={styles.introText}>
+          <p>Get to know us. Now.</p>
+        </div>
+        <div style={styles.imageBar}>
+          <img src={cat1}></img>
+          <img src={cat2}></img>
+          <img src={cat3}></img>
+        </div>
+        <div style={styles.aboutBody}>
+          <h2 style={styles.h2}>Brock</h2>
+          <p>A self-proclaimed data nerd with a distaste for baseless arguments and emotional decisions, Brock has always been about bringing facts to the gridiron. When he isn't serving up solutions at Sudokrew, a software shop in Hawaii, he's neck deep in deep learning and developing ways in which he can predict seemingly randoms events, such as sporting events and stock prices. If forced to name one thing he loves as much as football, it’d be Persian cats. </p>
```

Comment:
Let's add this information into a file called `aboutInfo.json` and import it into the `AboutUs` container. We can then run `.map` during component rendering and allow the content to be easily swapped out rather than hardcoded into the markup.